.editorconfig

```diff
@@ -0,0 +1,15 @@
+[*]
+charset=utf-8
+end_of_line=lf
+insert_final_newline=false
+indent_style=tab
+tab_width=4
+
+[{phpunit.xml.dist,*.jhm,*.xslt,*.xul,*.rng,*.xsl,*.xsd,*.ant,*.svg,*.tld,*.fxml,*.jrxml,*.xml,*.jnlp,*.wsdl}]
+indent_style=space
+indent_size=4
+
+[{arrconv54,*.hphp,*.phtml,*.module,*.php,*.php5,*.php4,*.inc}]
```

Comment:
Any reason for specifying extensions of a file, that doesn't even exist in the project?

Reply:
This was autogenerated by PHPStorm.  I see no reason for it not to be in

Reply:
I don't trust auto-generated stuff. Better to look deeper into file format itself and create content that would be most useful.

If by `autogenerated by PHPStorm` you mean `EditorConfig` plugin (see https://plugins.jetbrains.com/plugin/7294-editorconfig), then technically `.editorconfig` file generation way is how plugin developer sees it, not PhpStorm.